Cargo.lock

```diff
@@ -4,11 +4,11 @@ version = 3
 
 [[package]]
 name = "actix-codec"
-version = "0.5.1"
+version = "0.5.2"
```

Comment:
Did you run `cargo update`? There is suspiciously many changes in the `Cargo.lock`, I wouldn't expect that to be the case for the set of proposed changes.
If so, please rollback changes to it (as you could've accidentally repin some of the crypto dependencies) and make sure to only include changes relevant to the PR.

Reply:
i did to bump some crypto deps but ill specify those instead